test/package.json

```diff
@@ -19,7 +19,9 @@
     "undici": "^5.20.0",
     "socket.io": "^4.6.1",
     "socket.io-client": "^4.6.1",
-    "supertest": "^6.1.6"
+    "supertest": "^6.1.6",
```

Comment:
We should probably change `js/third_party` to be a mono repo, but we don't need to make that change in this PR.